doc/src/documentation/architecture/adr/adr003-test-as-prefix-for-tests.md

```diff
@@ -0,0 +1,25 @@
+# ADR-003 `test` as prefix for test case methods
+
+## Status
+
+<adr-status status='accepted' />
+
+## Context
+
+The code should have a certain uniformity. This also applies to the tests. From experience there were
```

Comment:
Past experiences show that there wrer different ...

Reply:
I also added the feedback of intelliJ: ~Past~ experiences show that there were different ...